libs/symphonia/participants.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
```

Comment:
will send you the output of this run which I think shows less than half deciding frequently for part in very low to med range... ie 1 - 30ish participants.